course_content/notebooks/dask_intro.ipynb

```diff
@@ -831,6 +831,94 @@
    "cell_type": "raw",
    "metadata": {},
    "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using `filter` as a Constraint for Iris Load with Dask\n",
+    "\n",
+    "We have seen how to load a subset of data by passing a `constraint` to `iris.load` when loading from `files`. We have also seen how to do this by passing lists of filenames and constraints to `delayed(iris.load)`, generating a `dask.bag`. Here we will use the `map` and `filter` functionality of `dask.bag` with `iris.load` to subset data using a phenomenon constraint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we should define what we mean when we talk about `map` and `filter`:\n",
+    "\n",
+    "`map(func, seq)`: A function which takes a function (`func`) and a sequence of inputs (`seq`) as arguments. It returns a sequence of outputs which are the result of applying `func` to each of the items in `seq`.\n",
```

Comment:
"A function which takes a function" is a little unclear! It's also technically not quite accurate, as `map` can take any **callable** as its first argument (including class methods, for example) - perhaps you could replace "function" with "callable" here therefore.